tests/test_set_override.py

```diff
@@ -197,10 +197,16 @@ def test_override_compute_constants(runner):
 
 def test_numbered_constants_must_be_overridden_by_number(runner):
     overrides = {"c": 23, "d": 24}
-    # This does absolutely nothing.  It doesn't even error.
-    assert [1, 2, 3, 0, 1, 2, 3, 0] == runner.run_test(
-        render=True, vertex_constants=overrides, fragment_constants=overrides
-    )
+    try:
+        # In naga, the bad constant is ignored.
+        # In the JS implementation, this throws an exception, which I think is the
+        # correct behavior.  So just in case this ever gets fixed, we accept either.
```

Comment:
Would you want to be notified when it gets fixed? Cause the test to fail?

Reply:
Yeah, the alternative is to let the test fail, and leave a long comment explaining what's going on and how to fix it. 

You're right, that's probably better.  I'll fix.